src/components/manga/reader/Page.tsx

```diff
@@ -31,9 +47,8 @@ function imageStyle(settings: IReaderSettings): any {
     return {
         display: 'block',
         marginBottom: settings.readerType === 'ContinuesVertical' ? '15px' : 0,
-        minWidth: '50vw',
-        width: '100%',
-        maxWidth: '100%',
+        width: dimensions.width < dimensions.height ? '100vw' : 'auto',
+        height: dimensions.width < dimensions.height ? 'auto' : '100vh',
```

Comment:
setting min and max width prevents images with too small or too big width to be shown correctly, don't remove them and instead handle the new case with right values.

Reply:
re-added min/max width. Removed `height:` since it might stretch image due to `minWidth` and fitting by `height`